app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,29 @@
 <!DOCTYPE html>
```

Comment:
A custom favicon would be a nice touch (nice to have vs. necessary, this would be a low priority item)

Reply:
Created issue #4 to address this later